source/includes/write/run-command.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+use MongoDB\Client;
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// start-runcommand
+$database = $client->accounts;
+$command = ['dbStats' => 1];
+
+$result = $database->command($command);
+// end-runcommand
+
+var_dump(json_encode($result->toArray()));
```

Comment:
Consider moving this within the code snippet, since the RST also shows example output. As-is, the code above wouldn't output anything.

Also, consider using `echo json_encode(...);`, optionally adding `, PHP_EOL;`. I don't think other tutorial snippets used `var_dump()` when printing JSON-encoded documents as strings.

Also, [`dbStats`](https://www.mongodb.com/docs/manual/reference/command/dbStats/#output) returns its result in single document, so we can unpack that and include a comment to explain why we're doing so. Assuming we rename `$result` to `$cursor`.

```
// dbStats returns a single document, so print the first document in the cursor
echo json_encode($cursor->toArray()[0]), PHP_EOL;
```

In the RST, this would be a good reason to mention that some commands return multiple results in a cursor, while others return single result documents. The PHP driver handles this by always returning a Cursor from the `command()` helper in the interest of providing a consistent API that handles _both_ variants.